modules/external-api/src/main/java/org/opencastproject/external/endpoint/GroupsEndpoint.java

```diff
@@ -67,7 +72,9 @@
 import javax.ws.rs.PathParam;
 import javax.ws.rs.Produces;
 import javax.ws.rs.QueryParam;
+import javax.ws.rs.WebApplicationException;
 import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
```

Comment:
Maybe instead import and use

javax.servlet.http.HttpServletResponse.SC_BAD_REQUEST
javax.servlet.http.HttpServletResponse.SC_CREATED

for sake of consistency.

Reply:
done